graylog2-web-interface/src/views/components/sidebar/SideBar.css

```diff
@@ -1,10 +1,74 @@
 :local(.sidebarContainer) {
-    margin-top: -15px;
+    padding-top: 20px;
+}
+
+:local(.sidebarNav) {
+    padding: 9px 0;
+    display: flex;
+    align-items: center;
+}
+
+:local(.sidebarNavFont) {
+    cursor: pointer;
+    font-size: 16px;
+    display: inline;
+    margin-top: 5px;
+    margin-left: 20px;
+    overflow: hidden;
+    white-space: nowrap;
+}
+
+:local(.selected) {
+   color: #FF3633;
+   background: #393939;
+}
+
+:local(.navContent) {
+    color: #666666;
+    background: #FFFFFF;
+    box-shadow:
+            inset 0px 13px 5px -10px #CCC,
+            inset 0px -13px 5px -10px #CCC;
+
+}
+
+:local(.contentOpen) {
+    padding: 20px;
+    max-height: 200px;
+    white-space: nowrap;
+}
+
+:local(.openFieldContent) {
+    white-space: nowrap;
+    padding: 20px;
+    position: absolute !important;
+    bottom: 20px;
+    height: calc(100vh - 314px);
+    max-height: calc(100vh - 314px);
```

Comment:
Is there no other way than doing pixel-based calculation?

I guess we can expect this to break/need adaption when we are changing anything that implicitly changes the size of the elements going into this calculation?